intersection_transit_maneuvering/src/intersection_transit_maneuvering.cpp

```diff
@@ -113,7 +113,11 @@ namespace intersection_transit_maneuvering
             /*Convert IT Straight*/
             if (maneuver.type == cav_msgs::Maneuver::INTERSECTION_TRANSIT_STRAIGHT)
             {
-                new_maneuver.lane_following_maneuver.parameters = maneuver.intersection_transit_straight_maneuver.parameters;
+                new_maneuver.lane_following_maneuver.parameters.maneuver_id = maneuver.intersection_transit_straight_maneuver.parameters.maneuver_id;
+                new_maneuver.lane_following_maneuver.parameters.planning_strategic_plugin = maneuver.intersection_transit_straight_maneuver.parameters.planning_strategic_plugin;
+                new_maneuver.lane_following_maneuver.parameters.planning_tactical_plugin = maneuver.intersection_transit_straight_maneuver.parameters.planning_tactical_plugin;
+                new_maneuver.lane_following_maneuver.parameters.presence_vector = maneuver.intersection_transit_straight_maneuver.parameters.presence_vector;
```

Comment:
Presence vector can be set to cav_msgs::ManeuverParameters::HAS_TACTICAL_PLUGIN to ensure the meta data fields are not accessed. Same for the other maneuver cases in this file.